java/jdbc/statement-interceptor/interceptor/gradle.properties

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2024, Oracle and/or its affiliates.
+#
+#   This software is dual-licensed to you under the Universal Permissive License
+#   (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+#   2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+#   either license.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#
+#
+
+
+jdbc.user=<CHANGEME>
```

Comment:
I find it confusing to find this configuration file in the interceptor. The interceptor shouldn't need to be configured to connect to a DB. A test would, but not the interceptor itself. I understand that this needed because of the main method in the Main class. But I think it should be removed. The test can require a DB configuration.